uniprot-datastore/src/main/java/org/uniprot/store/datastore/uniparc/UniParcStoreStep.java

```diff
@@ -0,0 +1,99 @@
+package org.uniprot.store.datastore.uniparc;
+
+import static org.uniprot.store.datastore.utils.Constants.UNIPARC_STORE_STEP;
+
+import net.jodah.failsafe.RetryPolicy;
+
+import org.springframework.aop.framework.Advised;
+import org.springframework.aop.support.AopUtils;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.listener.ExecutionContextPromotionListener;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.batch.item.ItemReader;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.uniprot.core.uniparc.UniParcEntry;
+import org.uniprot.core.xml.jaxb.uniparc.Entry;
+import org.uniprot.store.datastore.UniProtStoreClient;
+import org.uniprot.store.datastore.uniparc.config.UniParcStoreProperties;
+import org.uniprot.store.job.common.listener.LogRateListener;
+import org.uniprot.store.job.common.listener.WriteRetrierLogStepListener;
+import org.uniprot.store.job.common.writer.ItemRetryWriter;
+
+/**
+ * @author lgonzales
+ * @since 2020-03-03
+ */
+public class UniParcStoreStep {
+    private final StepBuilderFactory stepBuilderFactory;
+    private final UniParcStoreProperties uniParcStoreProperties;
+
+    @Autowired
+    public UniParcStoreStep(
+            StepBuilderFactory stepBuilderFactory, UniParcStoreProperties uniParcStoreProperties) {
+        this.stepBuilderFactory = stepBuilderFactory;
+        this.uniParcStoreProperties = uniParcStoreProperties;
+    }
+
+    @Bean(name = "uniParcStoreMainStep")
+    public Step uniParcStoreMainStep(
+            WriteRetrierLogStepListener writeRetrierLogStepListener,
+            @Qualifier("uniParcLogRateListener")
+                    LogRateListener<UniParcEntry> uniParcLogRateListener,
+            ItemReader<Entry> entryItemReader,
+            ItemProcessor<Entry, UniParcEntry> uniParcEntryProcessor,
+            ItemWriter<UniParcEntry> uniParcEntryItemWriter,
+            ExecutionContextPromotionListener promotionListener)
+            throws Exception {
+
+        return this.stepBuilderFactory
+                .get(UNIPARC_STORE_STEP)
+                .listener(promotionListener)
+                .<Entry, UniParcEntry>chunk(uniParcStoreProperties.getChunkSize())
```

Comment:
It is good to be consistent with the way we access this instance variable. 
I prefer to use this.<variableName>. But i am fine if we don't use it at all as well.
Either do `this. uniParcStoreProperties` or remove this from `this. stepBuilderFactory`